jwst/assign_wcs/nirspec.py

```diff
@@ -1696,6 +1696,129 @@ def gwa_to_ymsa(msa2gwa_model, lam_cen=None, slit=None, slit_y_range=None):
     return tab
 
 
+def get_transforms(input_model, slitnames, return_slits=False):
+
+    """
+    Returns a WCS object together with the transforms needed for all
+    slits.  This enables the code to avoid excessive deep copying of
+    of WCS objects.
+
```

Comment:
The first line of a Python docstring should be a short summary (<80 characters) and phrased as a command.  For example:
`Return a WCS object with necessary transforms for all slits.`
Any further clarification should be added in subsequent lines, after a blank line.  Can you please revise the docstring for this function and `nrs_wcs_set_input_lite`?

Reply:
Fixed.

Reply:
Thanks! Can you please also fix `nrs_wcs_set_input_lite`?

Reply:
Fixed (assuming you meant the docstring).

Reply:
I did, thanks!